cypress/e2e/login.cy.js

```diff
@@ -110,16 +110,17 @@ describe('Login tests', () => {
       cy.get('textarea[id=astarteToken]').paste(this.realm.infinite_token);
       cy.get('.btn[type=submit]').click();
 
-      cy.wrap(localStorage).its('session').then(JSON.parse).should('deep.eq', {
-        _version: 1,
-        realm: this.realm.name,
-        token: this.realm.infinite_token,
-        authUrl: null,
-      });
+      cy.getCookie('session')
+      .should('have.property', 'value', JSON.stringify({
+        "realm": this.realm.name,
+        "token": this.realm.infinite_token,
+        "authUrl": null,
+        "_version": 1
+      }).replaceAll("\"", "%22").replaceAll(",", "%2C"))
```

Comment:
We could use `decodeURIComponent` instead of implementing it with `replaceAll`
```suggestion
      cy.getCookie('session').its("value").then(decodeURIComponent).then(JSON.parse).should('deep.eq', {
          _version: 1,
          realm: this.realm.name,
          token: this.realm.infinite_token,
          authUrl: null
        })
```

Reply:
we might aswell change the test name to not include `localStorage`